chromadb/server/fastapi/__init__.py

```diff
@@ -565,8 +565,12 @@ async def create_collection(
         def process_create_collection(
             request: Request, tenant: str, database: str, raw_body: bytes
         ) -> CollectionModel:
+            json_decode = orjson.loads(raw_body)
+            # In 0.5.4 we added the configuration field to the CreateCollection model
+            # This field is optional, so we need to check if it exists in the json
+            if "configuration" not in json_decode:
+                json_decode["configuration"] = None
```

Comment:
is there a test/was there manual validation to double check this works with older clients?

Reply:
i did manual validation. we don't really have an easy way to plug in CI tests